MAINTAINERS.md

```diff
@@ -0,0 +1,53 @@
+# Istio Maintainers
+
+Here's a list of the fine folks that are the maintainers of the Istio project. They help guide the 
+technical direction, contribute to the growing code base, help take care of some of the bureaucratic duties 
+on the project, and help community members to merge their changes into the code base. Istio would not
+be what it is without these dedicated folks, so thanks!
+
+&nbsp; | Name | Company
+-------|------|--------
```

Comment:
Did you do this by hand or can this be automated if things change?

Reply:
I did it by hand, but I will definitely be automating this.